resources/dicts/events/new_cat/general.json

```diff
@@ -693,5 +693,116 @@
             "current_rep": ["any"],
             "changed": -1
         }
+    },
+    {
+        "event_id": "gen_new_cat_war_queen_with_litter",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 10,
+        "event_text": "A desperate queen from o_c_n slinks into c_n's territory, {PRONOUN/n_c:0/poss} newborn kits mewling fearfully. After a private discussion, both {PRONOUN/n_c:0/subject} and {PRONOUN/n_c:0/poss} kits are welcomed to the Clan.",
+        "new_cat": [
+            [
+                "new_name",
+                "can_birth",
+                "backstory:otherclan1",
+                "age:has_kits"
+            ],
+            [
+                "litter",
+                "parent:0",
+                "backstory:abandoned2",
+                "age:newborn"
+            ]
+        ],
+        "other_clan": {
+            "current_rep": ["any"],
+            "changed": -1
+        }
+    },
+    {
+        "event_id": "gen_new_cat_war_stolenkits1",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 10,
+        "event_text": "m_c leads a bold team to truly show those o_c_n mouse-brains which is the superior Clan. When dawn breaks, {PRONOUN/m_c/subject} and {PRONOUN/m_c/poss} team pad back to camp with a litter of mewling o_c_n kits in their jaws. <i>c_n</i> kits, now.",
+        "m_c": {
+            "status": [
+                "leader",
+                "deputy",
+                "warrior",
+                "medicine cat",
+                "apprentice",
+                "medicine cat apprentice",
+                "mediator",
+                "mediator apprentice"
+            ]
+        },
+        "new_cat": [
+            [
+                "can_birth",
+                "age:has_kits",
+                "clancat",
+                "meeting",
```

Comment:
hmm, i'm not sure how well this will actually work.. will it actually generate a clancat as an outsider and then handle it correctly? i'm not sure we have the functionality for that

Reply:
An idea to work around it, make it a loner/rogue with former clancat backstory, one of the ones of leaving the clan and not willing to talk about.

Then players can be like 'oh the cat left their clan to try and reclaim their kits' or some other story of why the parent left the clan in order to get back their kits. /idea